app/views/layouts/_header.html.erb

```diff
@@ -20,6 +20,11 @@
       <button type="button" role="button" class="govuk-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation" data-module="govuk-button">Menu</button>
       <nav>
         <ul id="navigation" class="govuk-header__navigation govuk-header__navigation--end" aria-label="Top Level Navigation">
+          <% if current_user.present? && current_user['associated_with_accredited_body'] %>
+            <li class="govuk-header__navigation-item">
+              <%= govuk_link_to "Notifications", notifications_path, class: "govuk-header__link", data:{qa: "notifications-link"} %>
```

Comment:
do we need the `data` attrribute? We are trying to move away from `data-qa` just for testing

Reply:
We also need an `active` state class on this link if the user is currently on the notifications page